src/components/molecules/atoms/TabItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface IProps {
+  isActive: boolean;
+  className?: string;
+  tabIndex: number;
+  onClick: (index: number) => void;
+}
+
+const TabItem: React.FC<IProps> = (props) => {
+  const {className} = props;
+
+  const handleClick = () => props.onClick(props.tabIndex);
+  return (
+    <li onClick={handleClick} className={className}>{props.children}</li>
```

Comment:
Again, this can be extracted into a `styled.li`. I don't know what the performance difference is, but as you're only really altering the styling, it might be more idiomatic.